docs/src/main/asciidoc/resteasy-reactive.adoc

```diff
@@ -9,7 +9,7 @@ include::./attributes.adoc[]
 :jaxrsapi: https://javadoc.io/doc/javax.ws.rs/javax.ws.rs-api/2.1.1
 :jaxrsspec: /specs/jaxrs/2.1/index.html
 :jdkapi: https://docs.oracle.com/en/java/javase/11/docs/api/java.base
-:mutinyapi: https://smallrye.io/smallrye-mutiny/apidocs
+:mutinyapi: https://javadoc.io/doc/io.smallrye.reactive/mutiny/1.6.0
```

Comment:
I would go with this one: https://javadoc.io/doc/io.smallrye.reactive/mutiny/latest/io.smallrye.mutiny/

We usually track the latest or close and we will avoid having to update the versions.

Reply:
It links to the 2.0.0-m1, are you sure ?

Reply:
Hmmmf. Yeah, not ideal. Let's keep it as it is.